contracts/modules/common/Utils.sol

```diff
@@ -50,7 +50,7 @@ library Utils {
     */
     function functionPrefix(bytes memory _data) internal pure returns (bytes4 prefix) {
         require(_data.length >= 4, "RM: Invalid functionPrefix");
-        // solium-disable-next-line security/no-inline-assembly
+
```

Comment:
I'm a bit unsure about removing all the assembly warnings from the code. In some cases, they may serve as an explicit reminder to check whether some new Solidity syntax has come out that removes the need for assembly. (This is such a case by the way: we could replace the following assembly code with the new array slicing syntax - something for a future PR).

Reply:
I've switched the `no-inline-assembly` rule to an error and explicitly ignored the existing 3 instances.

Reply:
There are a few more usages of `assembly` (in BaseModule, GuardianUtils, Utils).
Also, I think you forgot to include the `solhint-disable-next-line reason-string` for the `require(v == 27 || v == 28);` in Utils.

Reply:
I've switched missing or too long reason strings to warnings now as per https://github.com/argentlabs/argent-contracts/pull/137#discussion_r457459171

Reply:
There was also an issue running solhint in subfolders more than 1 level deep, the ** wildcard isn't working for some reason, I tried a couple of versions of `node` to check if it's that but without success. To fix it I've added a second run of solhint explicitly for the second level of folders. Obviously this won't work if we ever have more than 2 layers of folders but I can't see that happening.